modules/persistent-storage-local-tolerations.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// storage/persistent-storage/persistent-storage-local.adoc
+
+[id="local-tolerations_{context}"]
+= Using tolerations with Local Storage Operator Pods
+
+You can control which nodes the Local Storage Operator Pod runs on and prevent
+other workloads from using those nodes by using tolerations on the Pods.
+
+You apply tolerations to the Local Storage Operator Pod through the LocalVolume Custom Resource (CR)
+and apply taints to a node through the node specification. A taint on a node is a `key:value pair` that
+instructs the node to repel all Pods that do not tolerate the taint. Using a specific `key:value` pair
+that is not on other Pods ensures that only the Local Storage Operator Pod can run on that node.
```

Comment:
I believe the customer would have some workload that they want to reserve from general use. If that workload has local storage, then they need resources from the LSO to be created for their workload.

This is a nitpicky phrasing issue, because the goal isn't that **only** the LSO can run on the node. The goal is that the LSO can **also** run on the node, consuming local storage for whatever workload they're actually running.

Reply:
Text updated to match bolding (removed "only" and added "also").